src/DynamoCoreWpf/Services/UsageReportingManager.cs

```diff
@@ -36,6 +36,7 @@ public static void DestroyInstance()
             instance = null;
         }
 
+        public static bool IsShowGalleryAtStart { get; private set; }
```

Comment:
I think we better off rename this to `ShowGalleryAtStartUp` and move it into `PreferenceSettings`, and have the default value of `true`. Next time we can even implement a check box like *Show what's new gallery at start up*, which will be remembered in this setting.